samples/Gallery/UWP/SkiaSharpSample/MainPage.xaml.cs

```diff
@@ -52,19 +55,50 @@ public MainPage()
 				titlebar.ButtonHoverForegroundColor = Colors.White;
 			}
 
+			var _ = Dispatcher.RunAsync(CoreDispatcherPriority.Normal, async () => {
```

Comment:
This should probably not need to be used anymore. This was previously used because of the way CanvasKit was initializing, but it's not needed anymore (we're not using CanvasKit, we're using the actual skia lib)

Reply:
Awesome, I'll investigate.

Reply:
Removed. Thanks.